### ISSUE TITLE
Add railway and bus stop layers

### ISSUE THREAD

**Issue by mvl22** [cyipt/actdev-ui#72]
Add railway and bus stop layers

**Comment by mvl22** [comment]
Just to say I got this working pretty quickly this morning, but the layout will need adjusting before I can push this live, as the button lozenge is too long now on small screens.

![Screenshot 2021-03-19 at 21 08 17](https://user-images.githubusercontent.com/361423/111842412-563bee80-88f7-11eb-9785-792f0f453caa.png)

**Comment by Robinlovelace** [comment]
Could there be a combined 'PT' layer? I recall that would be tricky on the JS side but if possible could save a few mm. Look great though and seeing where the stops are gives a good first indication of PT provision (although not trip frequency which is relatively easy to change once the stops are there).

**Comment by mvl22** [comment]
> Could there be a combined 'PT' layer? I recall that would be tricky on the JS side

No, the layerviewer library does not have support for multiple distinct sources per layer, and adding support for multiple POI types on the server side API is not trivial.

> but if possible could save a few mm

Doesn't really solve the problem anyway, as the overflow would still be there.

Am considering how best to solve this UI issue.

**Comment by Siequnu** [comment]
Suggest a scrollable div entered within the map div.

**Comment by mvl22** [comment]
UI issue branched to new #83.
